Il2CppInterop.Runtime/DelegateSupport.cs

```diff
@@ -390,8 +388,7 @@ public override bool Equals(object obj)
 
     private class Il2CppToMonoDelegateReference : Object
     {
-        public IntPtr MethodInfo;
-        public Delegate ReferencedDelegate;
```

Comment:
I'm very happy to see this. One of the things in v2 that was bothering me is that I wasn't sure what to do with these fields. I'm introducing a breaking change that injected classes can't have managed state.